Sources/Tabs/ProfileCoordinator.swift

```diff
@@ -36,4 +39,25 @@ final class ProfileCoordinator: NSObject,
     func updateToken(_ tokenHolder: TokenHolder) {
         baseVc?.updateToken(tokenHolder)
     }
+
+    func openSignOutPage() {
+        let navController = UINavigationController()
+        let vm = SignOutPageViewModel(analyticsService: analyticsCenter.analyticsService) { [unowned self] in
+          do {
+            analyticsCenter.analyticsPreferenceStore.hasAcceptedAnalytics = false
+            analyticsCenter.analyticsService.denyAnalyticsPermission()
+            try? userStore.clearTokenInfo()
```

Comment:
Please at a TODO:  referencing https://govukverify.atlassian.net/browse/DCMAW-8933 above line 49, saying that the errors from userStore will be handled in that ticket.